operations/system_layout_service/Dump_SLS_Information.md

```diff
@@ -23,27 +23,13 @@ This procedure requires administrative privileges.
     }
     ```
 
-2.  Generate a private and public key pair.
+1. (`ncn-mw#`) Perform the SLS dump.
 
-    Execute the following commands to generate a private and public key to use for the dump.
+    The SLS dump will be stored in the `sls_dump.json` file. The `sls_dump.json` file is required to perform the SLS load state operation.
 
     ```bash
-    openssl genpkey -out private_key.pem -algorithm RSA -pkeyopt rsa_keygen_bits:2048
-    openssl rsa -in private_key.pem -outform PEM -pubout -out public_key.pem
-    ```
-
-    The above commands will create two files the private key private\_key.pem file and the public key public\_key.pem file.
-
-    Make sure to use a new private and public key pair for each dump operation, and do not reuse an existing private and public key pair. The private key should be treated securely because it will be required to decrypt the SLS dump file when the dump is loaded back into SLS. Once the private key is used to load state back into SLS, it should be considered insecure.
-
-3.  Perform the SLS dump.
-
-    The SLS dump will be stored in the sls\_dump.json file. The sls\_dump.json and private\_key.pem files are required to perform the SLS load state operation.
-
-    ```bash
-    curl -X POST \
+    curl -X GET \
```

Comment:
Can't this `curl` command be replaced with the craycli now? That now that https://github.com/Cray-HPE/craycli/pull/58 has merged?

Reply:
Ooh, that would be even nicer, if it's possible. If going that route, be sure to add a prerequisite about the Cray CLI being configured. Something like:

```markdown
- The Cray Command Line Interface is configured. See [Configure the Cray CLI](../configure_cray_cli.md).
```

Reply:
Yes, craycli can be used. I'll change the docs, and add the link to configure the cray cli.

Reply:
These changes have been made